src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -2494,7 +2494,7 @@ public static void GetHashCode_EmbeddedNull_ReturnsDifferentHashCodes()
         [InlineData("", "", StringComparison.OrdinalIgnoreCase, true)]
         [InlineData("123", 123, StringComparison.OrdinalIgnoreCase, false)] // Not a string
         [InlineData("\0AAAAAAAAA", "\0BBBBBBBBBBBB", StringComparison.OrdinalIgnoreCase, false)]
-        public static void Equals(string s1, object obj, StringComparison comparisonType, bool expected)
+        public static void Equals_Test(string s1, object obj, StringComparison comparisonType, bool expected)
```

Comment:
Changed the name of this purely because the analyser complained that this becomes an overload of `object.Equals(object)`. Shall I Suppress it instead?